tck/src/main/java/ee/jakarta/tck/data/framework/utilities/DatabaseType.java

```diff
@@ -21,14 +21,25 @@
  * This enum represents the configured DatabaseType based on the {@link TestProperty} databaseType
  */
 public enum DatabaseType {
-    UNKNOWN, //No database type was configured
-    RELATIONAL,
-    COLUMN,
-    DOCUMENT,
-    GRAPH,
-    KEY_VALUE;
+    UNKNOWN(Integer.MAX_VALUE), //No database type was configured
```

Comment:
I believe an UNKNOWN would be the data storage that won't fit on the previous ones, right?

For example, we are not talking about a time series database, such as [InfluxDB](https://www.influxdata.com/time-series-database/) or an Object database, such as [Eclipse Store](https://eclipsestore.io/).

In those cases, I believe it will fit better with a key-value database or don't have this option.
Make it mandatory or throw an exception.

Reply:
I don't think this needs to be mandatory.  If you don't configure the TCK with a database type then we assume your database supports all features tested.

The TCK cannot be written to perfectly run against every database, it is meant for certifying implementations.  If your Object database supports a feature list closer to a Key-Value database then run the TCK in that mode.

Reply:
Maybe define a relational database as default and remove this `UNKNOWN.`

Reply:
OTHER is fine.

Reply:
✅  Done